### PR TITLE
spark etl support kafka

### PR DESCRIPTION
**Pull request by ChengJie1053**
spark etl support kafka

### CODE REVIEW

File: linkis-engineconn-plugins/spark/pom.xml [hunk 1]
```diff
@@ -388,6 +388,17 @@
...
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-sql-kafka-0-10_${scala.binary.version}</artifactId>
+      <version>${spark.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.hadoop</groupId>
+          <artifactId>hadoop-client-runtime</artifactId>
+        </exclusion>
+      </exclusions>
```
Comment (peacewong) [inline linkis-engineconn-plugins/spark/pom.xml:400]: There is a kafka package in the Spark jars, which needs to be provided

Comment (peacewong) [reply linkis-engineconn-plugins/spark/pom.xml:400]: There is no reference in the code, you can directly delete this dependency

File: tool/dependencies/known-dependencies.txt [hunk 2]
```diff
@@ -491,7 +491,11 @@ snakeyaml-1.33.jar
 snappy-java-1.1.4.jar
 snappy-java-1.1.7.7.jar
 snappy-java-1.1.8.2.jar
+snappy-java-1.1.8.4.jar
 spark-redis_2.12-2.6.0.jar
+spark-sql-kafka-0-10_2.12-3.2.1.jar
+spark-tags_2.12-3.2.1.jar
+spark-token-provider-kafka-0-10_2.12-3.2.1.jar
```
[the remark this answers is not in the document]
Comment (ChengJie1053) [reply tool/dependencies/known-dependencies.txt:498]: ok

**Review by peacewong — APPROVED** [review]
LGTM.

**Pull request merged**